psi4/src/psi4/libsapt_solver/exch-ind30.cc

```diff
@@ -31,6 +31,11 @@
 #include "psi4/libpsio/psio.hpp"
 #include "psi4/libqt/qt.h"
 #include "psi4/libpsi4util/PsiOutStream.h"
+#include "psi4/lib3index/dfhelper.h"
```

Comment:
Why is this header needed?

Reply:
It wasn't. I copied and pasted too much.